ePBFT/src/pbft/libbyz/Replica.cpp

```diff
@@ -319,8 +319,11 @@ void Replica::init_state()
 
 void Replica::recv_start()
 {
-  // Compute session keys and send initial new-key message.
-  Node::send_new_key();
+  if (node_info.general_info.should_mac_message)
```

Comment:
Could you add a comment to where `should_mac_message` is defined to explain what this parameter changes?

Reply:
Do we ever need to mac messages now that CCF handles the node to node communication?